pkg/utils/log.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"runtime"
+
+	"github.com/stackrox/rox/pkg/logging"
+)
+
+var log = logging.LoggerForModule()
+
+// LogCaller logs the caller of this function
+// This helps trace where calls into deprecated features are coming from.
+// e.g. calls into RocksDB and BoltDB initialization
+func LogCaller(name string) {
```

Comment:
I don't think this is actually used anywhere cause you use debug.PrintStack() in the deprecated call

Reply:
fair point.  Would you prefer a call to this or `debug.PrintStack()`?  Easy enough to wipe this out if we don't need it.